packages/api/src/schema/api-schema.yaml

```diff
@@ -1845,13 +1913,105 @@ components:
           description: The error rate for the asset
         existsBeforeStart:
           type: number
-          description:
+          description: >-
             The percentage of sessions that existed before the asset started
             playing
+    create-asset-url:
+      type: object
+      required:
+        - url
+        - name
+      properties:
+        url:
+          type: string
+          example: $EXTERNAL_URL
+        name:
+          type: string
+          example: Example name
+    update-record-status:
+      type: object
+      required:
+        - record
+      properties:
+        record:
+          type: boolean
+          example: true
+    playback-info:
+      type: object
+      properties:
+        type:
+          type: string
+          example: vod
+        meta:
+          type: object
+          properties:
+            source:
+              type: array
+              items:
+                type: object
+                properties:
+                  hrn:
+                    type: string
+                    example: MP4
+                  type:
+                    type: string
+                    example: html5/video/mp4
+                  url:
+                    type: string
+                    example: >-
+                      https://asset-cdn.lp-playback.monster/hls/1bde4o2i6xycudoy/static360p0.mp4
+                  size:
+                    type: number
+                    example: 494778
+                  width:
+                    type: number
+                    example: 204
+                  height:
+                    type: number
+                    example: 360
+                  bitrate:
+                    type: number
+                    example: 449890
+      required:
+        - type
+        - meta
```

Comment:
This was long due, thanks! Would you mind replacing the custom type definition in [playback.ts](https://github.com/livepeer/livepeer-com/blob/b828376275e76abbddf0a524440fd08ee4a21bc8/packages/api/src/controllers/playback.ts#L33) with this one so we don't have to repeat it? (these types are auto-generated on the types.d.ts file, we already have some imports from there in that file)

Adding a suggestion here for a couple things that were missing from the original schema in typescript:
```suggestion
    playback-info:
      type: object
      properties:
        type:
          type: string
          example: vod
          enum: [live, vod, recording]
        meta:
          type: object
          required: [source]
          properties:
            source:
              type: array
              items:
                type: object
                required: [hrn, type, url]
                properties:
                  hrn:
                    type: string
                    example: MP4
                    enum: ["HLS (TS)", "MP4", "WebRTC (H264)"];
                  type:
                    type: string
                    example: html5/video/mp4
                    enum:
                     - html5/application/vnd.apple.mpegurl
                     - html5/video/mp4
                     - html5/video/h264
                  url:
                    type: string
                    example: >-
                      https://asset-cdn.lp-playback.monster/hls/1bde4o2i6xycudoy/static360p0.mp4
                  size:
                    type: number
                    example: 494778
                  width:
                    type: number
                    example: 204
                  height:
                    type: number
                    example: 360
                  bitrate:
                    type: number
                    example: 449890
      required:
        - type
        - meta
```